spec/features/google_tag_manager_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+
+RSpec.describe "Google Tag Manager" do
+  before :all do
```

Comment:
Beware of using before :all as it may cause state to leak between tests. If you are using rspec-rails, and use_transactional_fixtures is enabled, then records created in before :all are not automatically rolled back.